src/components/Menu.js

```diff
@@ -49,7 +53,8 @@ export function getMenuPlacement({
   placement,
   shouldScroll,
   isFixedPosition,
-}: PlacementArgs): MenuState {
+  theme,
+}: PlacementArgs) {
```

Comment:
@akx any reason you removed the return type?

Reply:
No idea! Restored.